docs/maui/converters/math-expression-converter.md

```diff
@@ -0,0 +1,95 @@
+---
+title: MathExpressionConverter - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: "The MathExpressionConverter is a converter that allows users to perform various math operations."
+ms.date: 03/16/2022
```

Comment:
Not hugely important, but we may want to update it a bit while we're still working on it